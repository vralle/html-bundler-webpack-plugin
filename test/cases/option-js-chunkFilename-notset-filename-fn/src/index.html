<!DOCTYPE html>
<html>
<head>
</head>
<body>
  <h1>Hello World!</h1>
  <script src="./main"></script>
  <!-- Note:
  here are injected not all generated split chunks.
  It is OK because other chunks are loaded dynamically at runtime from injected chunks.
  -->
</body>
</html>