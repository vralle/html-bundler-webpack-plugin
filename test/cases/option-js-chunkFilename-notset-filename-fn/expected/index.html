<!DOCTYPE html>
<html>
<head>
</head>
<body>
  <h1>Hello World!</h1>
  <script src="js/runtime.a689a002.js"></script>
<script src="js/803.2f7f93d4.js"></script>
<script src="js/main.9808bcf2.js"></script>
  <!-- Note:
  here are injected not all generated split chunks.
  It is OK because other chunks are loaded dynamically at runtime from injected chunks.
  -->
</body>
</html>